.github/scripts/clean-up.sh

```diff
@@ -7,10 +7,16 @@ set -o pipefail
 WORKDIR=$1
 PROJECT=$2
 
-oc delete -f ${WORKDIR}/hazelcast.yaml
-oc delete -f ${WORKDIR}/secret.yaml
-oc delete -f ${WORKDIR}/operator-rhel.yaml
-oc delete secret pull-secret
-oc delete project $PROJECT
+oc delete -f  ${WORKDIR}/hazelcast.yaml --wait=true
+oc delete -f  ${WORKDIR}/secret.yaml --wait=true
+oc delete -f  ${WORKDIR}/hazelcast-rbac.yaml --wait=true
+
+oc delete --wait=true rolebinding hazelcast-enterprise-operator
```

Comment:
Won't it just work with `oc delete -f bundle.yaml`?

Reply:
If there are CRs existing in other namespaces when we try to delete the CRD, the operation fails and script gets stuck deleting the CRD. So to solve this, we always apply the CRD in `bundle.yaml` whether it is already there or not, however we never delete the CRD to circumvent the situation I mentioned above. Those seperate `delete` commands are the result of this problem.

There is an issue about it https://github.com/elastic/cloud-on-k8s/issues/1701, at the bottom there is a link to another issue that show the temporary solution to the problem.